src/components/updatemenus/attributes.js

```diff
@@ -57,6 +57,37 @@ module.exports = {
         ].join(' ')
     },
 
+    openreverse: {
```

Comment:
Does this have to be a boolean? My initial thought is that it may be more clear if it's called something with `direction` or `dropdowndirection`/`expanddirection` then have options `up` / `down`

For me, the I think the `reverse` is a tiny bit confusing because I think of the options being in a reversed order rather than the dropdown opening up vs down.